test/parallel/test-util-inspect.js

```diff
@@ -1149,3 +1151,4 @@ if (typeof Symbol !== 'undefined') {
 }
 
 assert.doesNotThrow(() => util.inspect(process));
+/* eslint-enable accessor-pairs */
```

Comment:
I am not sure, but this line seems not needed since there is no code left for linting.

Reply:
Yeah, that was my thinking but I've noticed a bunch of our test files do this so I just followed the convention.

Reply:
Oh, maybe this is some over-caution for possible future code addition?

Reply:
Possibly. If someone does know, I would love to know the reason. Or maybe we're all just following the lead of the first person that did this... 😂

Reply:
I think the main reason is to try to limit the deactivation for any rule to the minimum required lines.

Reply:
Yes, in many tests such enabling comments are inserted in the middle of the code. But as the last line, they may be a bit confusing. However, they can have the same role as trailing commas in arrays and objects: safety device for future additions.